src/contracts/middleware/example/ECDSARegistry.sol

```diff
@@ -0,0 +1,260 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "../RegistryBase.sol";
+
+/**
+ * @title A Registry-type contract using aggregate BLS signatures.
```

Comment:
incorrect comment. there are similarly copy-pasted revert messages which still refer to "BLSRegistry"

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/7/commits/68f80a0c541b8ba06d37d8840a96fe9e4beee6e1